nims-notes-things/src/App.js

```diff
@@ -1,9 +1,14 @@
+import React from 'react';
+// import pencil from './penLogo.png';
 import "./App.css";
 import Form from "./Form";
 
 function App() {
   return (
     <div className="App">
```

Comment:
create a container component or styled component for this with App styles

Reply:
I didn't need that image imported there so i've taken it out